### DISCUSSION TITLE
Limit 'Next turn' navigation information

### DISCUSSION THREAD

**Discussion by g3ek** [osmandapp/OsmAnd#11675] [Q&A]
Good day to all!

We're planning on using OsmAnd for a group riding event. 
During a test I got frustrated that 'Next turn' widget in the top left signals even every slight curve of the road ahead.
I found this annoying because then I tended to miss important junctions which were immediately after this superfluous information.

Is there a setting I am missing?

**Comment by barteis** [comment]
I assume you are trying to follow a GPX.  Where was it created?

**Reply by g3ek** [reply]
It was created by me with a google map directions layer. 
Then exported to kml format and imported as a track.

**Reply by barteis** [reply]
Sounds like you've created a GPX using a Google map and Google's routing algorithm, as opposed to OpenStreetMap and OsmAnd's routing algorithm. OsmAnd can't convert the former into the latter, and so it navigates the GPX, but not the underlying map.  Which means OsmAnd can identify turns, but not intersections or street names.  

You'll get true turn-by-turn navigation by creating the GPX using OsmAnd's Plan a route function. This video tutorial might help: https://youtu.be/bYQpp9UWqlI
